packages/core/src/__tests__/__e2e__/banner/Banner.cy.tsx

```diff
@@ -13,10 +13,10 @@ describe("GIVEN a Banner", () => {
   it("THEN should render status", () => {
     cy.mount(<StatusesPrimary />);
 
-    cy.findByTestId("InfoSolidIcon").should("exist");
-    cy.findByTestId("SuccessTickIcon").should("exist");
-    cy.findByTestId("WarningSolidIcon").should("exist");
-    cy.findByTestId("ErrorSolidIcon").should("exist");
+    cy.findByRole("img", { name: "info" }).should("exist");
```

Comment:
I'm not against the test changes but it's worth highlighting these don't test the same thing, one is testing the actual icon and the other is testing the accessible name of the icon. Which should be fine paired with our visual regression testing.